airflow/providers/amazon/aws/hooks/redshift_data.py

```diff
@@ -53,6 +53,7 @@ def execute_query(
         database: str,
         sql: str | list[str],
         cluster_identifier: str | None = None,
+        workgroup_name: str | None = None,
```

Comment:
Should we add this new parameter to the end of this function? Some users might pass positional arguments.

Reply:
Good point Lee, I'll make it happen.

Reply:
Thank you, addressed in 56cc6d8ea13031ca610250735c8b0071065b233f